front/src/applications/stdcm/components/StdcmResults/StdcmResultsTable.tsx

```diff
@@ -43,7 +44,7 @@ const StcdmResultsTable = ({
             <th>{t('passageStop')}</th>
             <th>{t('startStop')}</th>
             <th className="weight">{t('weight')}</th>
-            <th>{t('refEngine')}</th>
+            <th>{capitalizeFirstLetter(t('refEngine'))}</th>
```

Comment:
Instead of post-processing the translated strings, could we update the strings in the translation files instead?

Reply:
since it only used once, yes we can

Reply:
fixed